### PR TITLE
Component/action link

### LINKED ISSUE #2: Actionlink Component
Bildschirmfoto 2021-11-15 um 11 47 26

### PR DESCRIPTION
**Pull request by derjubu**
Fixes #2 
- Add button
- Add SVG
- Add styling
Bildschirmfoto 2021-11-16 um 14 49 01

### CODE REVIEW

File: src/app/components/Actionlink/DoubleRight.svg [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (phlgr) [inline src/app/components/Actionlink/DoubleRight.svg:1]: This file should not be committed

File: src/app/components/Icons/ActionlinkIcon.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+export default function ActionLinkIcon(
+  props: React.SVGProps<SVGSVGElement>
+): JSX.Element {
+  return (
+    <svg
+      width="24"
+      height="24"
+      viewBox="0 0 24 24"
+      fill="none"
+      {...props}
```
Comment (phlgr) [inline src/app/components/Icons/ActionlinkIcon.tsx:12]: Move this down

File: src/app/components/Actionlink/Actionlink.tsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import ActionLinkIcon from '../Icons/ActionlinkIcon';
+
+type ActionlinkProps = {
+  url: string;
+  text: string;
+};
+
+function ActionLink({ url, text }: ActionlinkProps): JSX.Element {
```
Comment (phlgr) [inline src/app/components/Actionlink/Actionlink.tsx:10]: The component should display the Arrows when needed and not on hover

File: src/app/components/Actionlink/Actionlink.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ActionLink from './Actionlink';
+
+export default {
+  title: 'Component/Actionlink',
+  component: ActionLink,
+};
+
+export const Imdb = (): JSX.Element => (
+  <ActionLink url={'http://www.imdb.com'} text={'To imdb'} />
```
Comment (phlgr) [inline src/app/components/Actionlink/Actionlink.stories.tsx:10]:
```suggestion
  <ActionLink url="http://www.imdb.com" text="To IMDB" />
```

**Review by phlgr — CHANGES_REQUESTED** [review]
I would like to see some more Stories to see how the ActionLink behaves when the text is longer

**Review by marwinburesch — APPROVED** [review]
💯

**Pull request merged**